src/colorizer/shaders/colorize_RGBA8U.frag

```diff
@@ -144,7 +144,7 @@ vec4 getBackdropColor(vec2 sUv) {
   return vec4(backdropRgb, backdropColor.a);
 }
 
-vec4 getObjectColor(vec2 sUv) {
+vec4 getObjectColor(vec2 sUv, float opacity) {
```

Comment:
Thought: could use the `objectOpacity` uniform rather than an argument here? Honestly don't know which is cleaner